requirements-dev.txt

```diff
@@ -11,5 +11,5 @@ black; python_version >= '3.6'
 attrs>=17.4.0
 pre-commit
 # Until https://github.com/spulec/moto/issues/1793 is resolved
-boto3 < 1.8.0
+#boto3 < 1.8.0
```

Comment:
I think we still need this or tests can fail until the issue above is resolved. (though botocore is the real source of the issue on the line below). I'd prefer to leave them both in until the fix referenced above is resolved.